client/src/pages/Home.js

```diff
@@ -1,27 +1,9 @@
-import { useEffect, useState } from "react";
-import { Link } from "react-router-dom";
 
+import { Link } from "react-router-dom";
+import { Button } from "react-bootstrap";
 import "./Home.css";
-import logo from "./logo.svg";
 
 export function Home() {
-	const [message, setMessage] = useState("Loading...");
```

Comment:
Hi Christian, I dont think we should delete this code as it is linked to the api which is used in the app.js file and linked to my end points done for the student registration form

Reply:
ohh   i did not  know that i thought it was just skeleton code